Answer FAQ about differences between releases (#140)
AlexsLemonade/refinebio-docs: docs/faq.md

When using quantile normalization, the expression value a gene is assigned in a particular sample depends on the _rank_ of that gene.
If a user download different datasets, which may have different numbers of genes, it's possible then that the same gene in the same sample would have a different expression value between them.

#### Why do I get a limited number of genes back when I aggregate samples from different experiments?

When aggregating samples, we retain _only_ the genes present in _every sample_.
Different microarray platforms will often measure different sets of genes. 
RNA-seq samples in refine.bio can be quantified using different transcriptome indices, which may be built using different Ensembl releases and therefore include different sets of genes.

> Thus, when aggregating samples from multiple experiments, it is not uncommon for some genes measured on a platform to be dropped.

Reviewer: I reversed platform/experiment here for what I think is improved clarity. I could be wrong.
```suggestion
Thus, when aggregating samples from multiple platforms, it is not uncommon for some genes measured in individual experiments to be dropped.
```

refine.bio will sometimes obtain the metadata (e.g., sample title or experimental protocol) associated with a sample but the raw or submitter processed expression data files are in a format that we can not process.
We do not allow you to add these samples to your dataset because we can not deliver expression values.

> #### Why do the genes differ between different RNA-seq experiments from the same organism?

Author: What do you think of `Why do the genes included in RNA-seq experiments change between experiments from the same organism?` ?

> #### Why do the genes differ between different RNA-seq experiments from the same organism?

Reviewer: Not bad. I don't love "experiments... experiments" How about: `Why do the genes included in RNA-seq datasets change between experiments from the same organism?`

> #### Why do the genes differ between different RNA-seq experiments from the same organism?

Author: I also don't love "experiments...experiments" _but_ we use "dataset" to refer to "everything you put in your shopping cart and asked to download" and "experiment" to refer to a study (project or series accession for SRA and GEO, respectively) so I am concerned about introducing an inconsistency in how we use terminology to make this "sound better."

You may find that there are a small proportion of genes that differ between different RNA-seq from the same organism when you download data from refine.bio.

> The difference in gene sets can be detected when downloading mulitple experiments as part of separate dataset downloads or when aggregating by experiment.

Author: I rewrote the answer to this question to be less microarray-centric and changed the header (`platforms` -> `experiments`).
